modules/social_features/social_landing_page/social_landing_page.libraries.yml

```diff
@@ -10,11 +10,6 @@ section.introduction:
     theme:
       css/social_landing_page.section.introduction.css: {}
 
-section.featured:
-  css:
-    theme:
-      css/social_landing_page.section.featured.css: {}
-
```

Comment:
Removing this library is a backwards-incompatible change and may break other modules or themes overriding/extending the library. Ideally we mark the library as deprecated but keep using it for now.

Reply:
I did the same as you did [here](https://github.com/goalgorilla/open_social/pull/1766/files#diff-37d3a37e02d24dff66f6a231753c2c3d)

Reply:
Moving this one is fine as for me. As Oleksandr said it is the same as we did with featured items before.
But maybe we need to create Change records for both: https://www.drupal.org/list-changes/social